### PR TITLE
Feature/feedback widget

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

File: moj-node/server/repositories/analytics.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ function analyticsRepository(httpClient) {
       el: label,
     };
 
-    if (value !== undefined) {
+    if (value) {
```
Comment (psoleckimoj) [inline moj-node/server/repositories/analytics.js:15]: Can you change this to match the branch as this doesn't work, needs to check for undefined :)

File: moj-node/server/routes/games.js [hunk 2]
```diff
@@ -26,9 +26,6 @@ const createGamesRouter = ({ analyticsService, logger }) => {
 
     return res.render('pages/games/chess', {
       title: 'Chess',
```
Comment (psoleckimoj) [inline moj-node/server/routes/games.js:28]: Not sure why all these have changed?

File: moj-node/server/repositories/analytics.js (hunk 1 above)
Comment (LucasCairns) [reply moj-node/server/repositories/analytics.js]: Removed the commit

File: moj-node/server/routes/games.js (hunk 2 above)
Comment (LucasCairns) [reply moj-node/server/routes/games.js:28]: As discussed

**Pull request merged**